pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -24,121 +24,92 @@ import (
 
 	"github.com/vmware-tanzu/antrea/pkg/agent/config"
 	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
-	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
 	"github.com/vmware-tanzu/antrea/pkg/agent/util/sysctl"
 )
 
-var _ ConnTrackDumper = new(connTrackDumper)
+// connTrackSystem implements ConnTrackDumper. This is for linux kernel datapath.
+var _ ConnTrackDumper = new(connTrackSystem)
 
-type connTrackDumper struct {
+type connTrackSystem struct {
 	nodeConfig  *config.NodeConfig
 	serviceCIDR *net.IPNet
-	connTrack   ConnTrackInterfacer
+	connTrack   NetFilterConnTrack
 }
 
-func NewConnTrackDumper(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet, conntrack ConnTrackInterfacer) *connTrackDumper {
-	return &connTrackDumper{
+func NewConnTrackSystem(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet) *connTrackSystem {
+	// Ensure net.netfilter.nf_conntrack_acct value to be 1. This will enable flow exporter to export stats of connections.
+	// Do not handle error and continue with creation of interfacer object as we can still dump flows with no stats.
+	// If log says permission error, please ensure net.netfilter.nf_conntrack_acct to be set to 1.
```

Comment:
I know this is not introduced by this PR, but this sentence confuses me:
> 	If log says permission error, please ensure net.netfilter.nf_conntrack_acct to be set to 1.

What could cause a permission error here? Can this happen when Antrea Agent is run "normally" (privileged container)? Given that a user is probably not going to see this comment, who is this addressed to?

Reply:
I have seen this error only on Kind clusters. If it is a privileged container, yes then this is irrelevant. May be I could remove this?